src/marten/db/query/raw_node.cr

```diff
@@ -0,0 +1,40 @@
+module Marten
+  module DB
+    module Query
+      class RawNode < Node
+        getter statement
+        getter params
+
+        def initialize(
+          @statement : String,
+          @params = [] of ::DB::Any,
+          @children = [] of Node,
+          @connector = SQL::PredicateConnector::AND,
+          @negated = false
```

Comment:
Similarly, I think we can avoid making it possible to initialize raw node objects with `negated: true` since we don't need this capability for raw subqueries.